src/implementation/blascontract.jl

```diff
@@ -55,8 +55,8 @@ function _blas_contract!(C, A, pA, B, pB, pAB, α, β, backend, allocator)
         tensoradd!(C, C_, pAB, false, α, β, backend, allocator)
         tensorfree!(C_.parent, allocator)
     end
-    flagA || tensorfree!(A_.parent, allocator)
-    flagB || tensorfree!(B_.parent, allocator)
+    flagA || tensorfree!(A_, allocator)
+    flagB || tensorfree!(B_, allocator)
```

Comment:
What was the reason for this change? Will this still work if `A_` is a `StridedView` over a `PtrArray`? Or do we need a `tensorfree!(A::StridedView, allocator) = tensorfree!(A.parent, allocator)` definition?

Reply:
Oh I meant to revert this as well,  I changed things around a bit when I wanted to do the normalization of the Strided parent here, but I'll change it back tomorrow.